pkg/clusterlink/controllers/node/node_controller.go

```diff
@@ -109,20 +107,11 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 			},
 		},
 	}
-	cluster, err := r.ClusterLinkClient.KosmosV1alpha1().Clusters().Get(ctx, r.ClusterName, metav1.GetOptions{})
-	if err != nil {
-		klog.Errorf("get cluster %s err: %v", r.ClusterName, err)
-		return reconcile.Result{Requeue: true}, nil
-	}
 
-	err = CreateOrUpdateClusterNode(r.ClusterLinkClient, clusterNode, func(n *clusterlinkv1alpha1.ClusterNode) error {
+	err := CreateOrUpdateClusterNode(r.ClusterLinkClient, clusterNode, func(n *clusterlinkv1alpha1.ClusterNode) error {
 		n.Spec.NodeName = node.Name
 		n.Spec.ClusterName = r.ClusterName
-		n.Spec.InterfaceName = interfacepolicy.GetInterfaceName(cluster.Spec.ClusterLinkOptions.NICNodeNames, node.Name, cluster.Spec.ClusterLinkOptions.DefaultNICName)
```

Comment:
Can agent perform ip maintenance on clusternode in 2 modes: auto-detect and manual configuration?